### ISSUE TITLE
Sort shortcuts

### ISSUE THREAD

**Issue by friday** [Ulauncher/Ulauncher#1008]
### Description

Add the ability to rearrange the order of the shortcuts with drag and drop in the shortcut preferences and get the same order in the default search suggestions if there are more than one default search shortcuts.

### Communication guidelines

- [X] I have read and agree with Ulauncher's communication guidelines and checked the [existing enhancement requests](https://github.com/Ulauncher/Ulauncher/labels/enhancement) before suggesting this.

**Comment by friday** [comment]
> Is this still something you'd want implemented? I think I could get it working using [SortableJS](https://github.com/SortableJS/Sortable)

No, since we want to replace the whole Preferences UI soon the new one would be written in a completely different way.

**Comment by iago-lito** [comment]
Alternately:

- Sort them alphabetically (resulting in *less* scrolling required and no need to sort manually).

Or 

- add new behaviour to the ⚙️ button behaviour on the right of the main search bar:
  -  fuzzy-search for the shortcut as usual
  - instead of pressing 'enter', click ⚙️  to directly open that very shortcut config panel.

(resulting in *no* scrolling required and no need to sort at all)

**Comment by friday** [comment]
I haven't thought about this in ages. I had to process a bit to remind myself why the manual sorting I suggested could make sense, and I think it was because of the fallback search behavior. It makes sense for the user to be able to decide the render order.

The order in the preferences view is secondary to this I think. But if you have a lot of shortcuts you could need them sorted alphabetically or a filter/search there too I guess. And this is also easier to fix. The problem is these two directions may not be compatible.

**Comment by iago-lito** [comment]
> because of the fallback search behavior. It makes sense for the user to be able to decide the render order.

Agreed.

> if you have a lot of shortcuts you could need them sorted alphabetically or a filter/search there too

Indeed. I do have a lot of shortcuts and it is currently a pain to find the one I want to update. Alphabetical sorting would be nice. Filter/search would be better. And my opinion is that.. well, `Ulauncher` is _already_ a filter-search over shortcuts ;) Thus my suggestion to just (re)use the main interface as(=which-is-already) a shortcut selector.

*E.g.*: to edit shortcut `gh` to `github.com` I'll just trigger ulauncher, type in `gh` to see it appear then instead of pressing `<enter>` I'd click the ⚙️ button to open the `github.com` config panel immediately :)
